examples/classification/configs/mixed_precision/squeezenet1_1_imagenet_mixed_int_manual_staged.json

```diff
@@ -42,17 +42,17 @@
-                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[10]/ReLU[expand3x3_activation]/RELU_0]"],
-                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[10]/ReLU[squeeze_activation]/RELU_0]"],
-                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[11]/ReLU[expand1x1_activation]/RELU_0]"],
-                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[11]/ReLU[expand3x3_activation]/RELU_0]"],
-                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[11]/ReLU[squeeze_activation]/RELU_0]"],
-                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[12]/ReLU[expand1x1_activation]/RELU_0]"],
-                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[12]/ReLU[expand3x3_activation]/RELU_0]"],
-                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[9]/ReLU[expand1x1_activation]/RELU_0]"],
-                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[9]/ReLU[expand3x3_activation]/RELU_0]"],
-                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[9]/ReLU[squeeze_activation]/RELU_0]"],
+                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[10]/ReLU[expand1x1_activation]/RELU_0|OUTPUT]"],
+                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[10]/ReLU[expand3x3_activation]/RELU_0|OUTPUT]"],
+                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[10]/ReLU[squeeze_activation]/RELU_0|OUTPUT]"],
+                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[11]/ReLU[expand1x1_activation]/RELU_0|OUTPUT]"],
+                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[11]/ReLU[expand3x3_activation]/RELU_0|OUTPUT]"],
+                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[11]/ReLU[squeeze_activation]/RELU_0|OUTPUT]"],
+                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[12]/ReLU[expand1x1_activation]/RELU_0|OUTPUT]"],
+                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[12]/ReLU[expand3x3_activation]/RELU_0|OUTPUT]"],
+                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[9]/ReLU[expand1x1_activation]/RELU_0|OUTPUT]"],
+                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[9]/ReLU[expand3x3_activation]/RELU_0|OUTPUT]"],
+                    [4, "ModuleDict/AsymmetricQuantizer[SqueezeNet/Sequential[features]/Fire[9]/ReLU[squeeze_activation]/RELU_0|OUTPUT]"],
```

Comment:
Is it related to this PR? Looks like there's a different issue with scopes. Moreover, there should be some changes in tests, otherwise this bug is not covered by them.

Reply:
Indeed, this PR also refactors some of the quantization code (namely removes `InsertionInfo` from the majority of places and replaces it with `InsertionPoint`), and this uncovered a bug where the post-hook activation quantizer names in the model's state dict were not stored with a fully qualified name. It is not, however, a bug per se, but rather a convention that has changed. The pull request also adds a tool to convert the existing checkpoints to the new format, and I will do this for our current checkpoint set and reupload them.